src/validation/Validator.ts

```diff
@@ -332,6 +341,34 @@ export class Validator {
         return value instanceof Boolean || typeof value === "boolean";
     }
 
+
+    /**
+    * Checks if a given value is a latitude.
+    */
+    
+    isLatLong(value: any): boolean {
+
+        return this.validatorJs.isLatLong(value);
+    }
+
+    /**
+    * Checks if a given value is a latitude.
+    */
+
+
```

Comment:
Extra lines between comment and function - why?

Reply:
just wanted to make it look clear on my ide

Reply:
Sorry but extra empty lines between comment and function related directly to comment do not look clearly. It is pointless :) Check another code in `class-validator` and try to use same formatting as we use before in our code base.